addons/point_of_sale/models/account_move.py

```diff
@@ -10,6 +10,18 @@ class AccountMove(models.Model):
     pos_order_ids = fields.One2many('pos.order', 'account_move')
     pos_payment_ids = fields.One2many('pos.payment', 'account_move_id')
 
+    @api.depends('move_type', 'line_ids.amount_residual')
```

Comment:
You don't need to repeat depends when inheriting computes

Reply:
thanks & updated